src/audio/kpb.c

```diff
@@ -51,10 +51,15 @@
 static const struct comp_driver comp_kpb;
 
 LOG_MODULE_REGISTER(kpb, CONFIG_SOF_LOG_LEVEL);
-
+#if CONFIG_IPC_MAJOR_3
```

Comment:
Instead of #if CONFIG_IPC_MAJOR_3 please make this as default option (#else) and the new UUID should be #if IPC4 is selected

Reply:
Done